commands/command_e2e_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestModifyCommandCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	// create new command
+	newCmd := &model.Command{
+		CreatorId: s.th.BasicUser.Id,
+		TeamId:    s.th.BasicTeam.Id,
+		URL:       "http://nowhere.com",
+		Method:    model.COMMAND_METHOD_POST,
+		Trigger:   "trigger",
+	}
+
+	command, _ := s.th.SystemAdminClient.CreateCommand(newCmd)
+	copy := command
+	originalURL := newCmd.URL
+	index := 0
+	s.RunForAllClients("modifyCommandCmdF", func(c client.Client) {
+		printer.Clean()
+
+		command = copy
+		s.Require().Equal(command.URL, originalURL)
+
+		// Reset the cmd and parse to force Flag.Changed to be true.
+		cmd := CommandModifyCmd
+		cmd.ResetFlags()
+		addCommandFieldsFlags(cmd)
+		url := fmt.Sprintf("%s-%d", command.URL, index)
+		index++
+		err := cmd.ParseFlags([]string{
+			command.Id,
+			"--url=" + url,
+		})
+		s.Require().Nil(err)
+
+		err = modifyCommandCmdF(s.th.SystemAdminClient, cmd, []string{command.Id})
+		s.Require().Nil(err)
+		s.Len(printer.GetLines(), 1)
+		s.Len(printer.GetErrorLines(), 0)
+
+		check := getCommandFromCommandArg(s.th.SystemAdminClient, command.Id)
+		s.Require().Equal(check.URL, url)
+	})
+
+	s.RunForAllClients("modifyCommandCmdF for command that does not exist", func(c client.Client) {
+		printer.Clean()
+		cmd := &cobra.Command{}
+
+		err := modifyCommandCmdF(s.th.SystemAdminClient, cmd, []string{"nothing"})
```

Comment:
Same client suggestion here:

```suggestion
		err := modifyCommandCmdF(c, cmd, []string{"nothing"})
```